### PR TITLE
Reusable recipes

### PR DESCRIPTION
**Pull request by thomasstvr**
I made all the separations of the functions as discussed but I'm sure it will require a bit of reorganization. Let me know what you think.

### CODE REVIEW

File: wrangles/recipe.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 from . import recipe_wrangles as _recipe_wrangles
 from . import connectors as _connectors
 from .config import no_where_list
+from wrangles.data import model_content as _model_content
```
Comment (ChrisWRWX) [inline wrangles/recipe.py:19]: You should use the "from . import x as y" style import. The wrangles.data style can have issues when it's a package, depending on the circumstances.

File: wrangles/console.py [hunk 2]
```diff
@@ -42,4 +33,4 @@ def recipe():
         variables = {}
 
     # Run the recipe
-    _recipe.run(args.recipe.read(), functions=functions, variables=variables)
+    _recipe.run(args.recipe.read(), variables=variables)
```
[the remark this answers is not in the document]
Comment (thomasstvr) [reply wrangles/console.py:36]: This part got moved to recipe.py, functions still work in the console

[the remark this answers is not in the document]
Comment (ChrisWRWX) [reply wrangles/console.py:36]: Yeah, the code above makes sense to be removed - but it's not calling _recipe.run() with functions, so has no way of knowning what functions were requested by the console command...

[the remark this answers is not in the document]
Comment (thomasstvr) [reply wrangles/console.py]: Where does the terminal pull this from? I tried breaking it to see if it was getting it locally but it isn't.

**Pull request closed without merging**